src/components/Notifications/Notifications.tsx

```diff
@@ -0,0 +1,49 @@
+import React, { useContext, useEffect, useState } from 'react';
+import cn from 'classnames';
+import { ErrorTypes } from '../../types/ErrorTypes';
+import { TodosContext } from '../TodosProvider';
+import { GetErrorMessage } from '../../utils/GetErrorMessage';
+
+export const Notifications: React.FC = React.memo(() => {
+  const [isHidden, setIsHidden] = useState(true);
+  const { errorType, errorTypeHandler } = useContext(TodosContext);
+  const errorMessage = GetErrorMessage(errorType);
```

Comment:
function name should start from small letter

Reply:
fixed